internal/cli/delete.go

```diff
@@ -0,0 +1,156 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/futurice/jalapeno/internal/cli/option"
+	"github.com/futurice/jalapeno/pkg/recipe"
+	"github.com/futurice/jalapeno/pkg/ui/colors"
+	"github.com/gofrs/uuid"
+	"github.com/spf13/cobra"
+)
+
+type deleteOptions struct {
+	SauceID string
+	All     bool
+
+	option.Common
+	option.WorkingDirectory
+}
+
+func NewDeleteCmd() *cobra.Command {
+	var opts deleteOptions
+	var cmd = &cobra.Command{
+		Use:   "delete [SAUCE_ID]",
```

Comment:
Just a note: I didn't mention this in the issue description, but at the moment there is no clean way to get the sauce ID other than manually looking it up from the `sauces.yml` file. Later we could have a command to list all the sauces for easy access.

We should document how to use this command at the end of the [`usage`](https://github.com/futurice/jalapeno/blob/main/docs/site/docs/usage.mdx?plain=1) page.